tutorials/colors/colormaps.py

```diff
@@ -199,30 +199,29 @@
 # First, we'll show the range of each colormap. Note that some seem
 # to change more "quickly" than others.
 
-nrows = max(len(cmap_list) for cmap_category, cmap_list in cmaps.items())
 gradient = np.linspace(0, 1, 256)
 gradient = np.vstack((gradient, gradient))
 
 
 def plot_color_gradients(cmap_category, cmap_list, nrows):
-    fig, axs = plt.subplots(nrows=nrows)
-    fig.subplots_adjust(top=0.95, bottom=0.01, left=0.2, right=0.99)
-    axs[0].set_title(cmap_category + ' colormaps', fontsize=14)
+    fig, axes = plt.subplots(nrows=nrows)
+    fig.subplots_adjust(top=(0.0522*nrows), bottom=0.01, left=0.2, right=0.99)
```

Comment:
The white space is because the figure is larger than the required space for the information being plotted.  Just changing the margins changes where the whitespace is, but does not get rid of it.  

You could change the figure size, but then the physical size of the subplots would change.  You would need to do some work to make the colorbars the same physical size, though it is of course possible.  

sphinx-gallery does not have an option to call `bbox_inches='tight'`: https://github.com/sphinx-gallery/sphinx-gallery/issues/726  You can make static pngs that then we load for display, bypassing sphinx gallery.